File: Java/src/Iniciante/Solucao_1101.java
package Iniciante;
import java.util.ArrayList;
import java.util.Scanner;

public class Solucao_1101 {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);

        while(true) {
            int soma = 0;
            ArrayList<Integer> numeros = new ArrayList<>();
            int num, num2, maior, menor;

            num = input.nextInt();
            num2 = input.nextInt();

            if (num <= 0 || num2 <= 0) {
                break;
            }else {
                if (num > num2) {
                    maior = num;
                    menor = num2;
                } else {
                    maior = num2;
                    menor = num;
                }
                for (int i = menor; i <= maior; i++) {
                    numeros.add(i);
                    soma += i;
                }

                for (Integer numero : numeros) {
                    System.out.print(numero + " ");
                }
                System.out.printf("Sum=%d%n", soma);

            }
        }
            input.close();
    }
}

File: Python/Iniciante/Solucao_1101.py

while True:
    soma = 0
    lista = []
    n,m = map(int,input().split())
    if n <= 0 or m <= 0:
        break
    if n > m:
        for i in range(m, n+1):
            lista.append(i)
            soma += i
    else:
        for i in range(n, m+1 ):
            lista.append(i)
            soma += i
    lista_str = map(str,lista)
    lista_str = " ".join(lista_str)

    print(f"{lista_str} Sum={soma}")
